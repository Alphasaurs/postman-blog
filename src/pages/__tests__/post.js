// import React from "react"
// import renderer from "react-test-renderer"
// import BlogPostTemplate from "../../templates/post"
// import { StaticQuery } from "gatsby"

// beforeEach(() => {
//   StaticQuery.mockImplementationOnce(({ render }) =>
//     render({
//       wpgraphql
//     })
//   )
// })


// this stores the graphql call in a snapshot, does not test rendering?
import postPageQuery from '../../templates/post'
describe("BlogPostTemplate", () => {
  it("it should be the correct query", () => {
    // const tree = renderer.create(<BlogPostTemplate />).toJSON()
    expect(postPageQuery).toMatchSnapshot()
  })
})