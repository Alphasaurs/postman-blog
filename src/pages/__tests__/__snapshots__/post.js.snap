// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPostTemplate it should be the correct query 1`] = `[Function]`;
